src/js/stores/DCOSStore.js

```diff
@@ -25,7 +25,9 @@ class DCOSStore extends EventEmitter {
     });
 
     this.data = {
-      marathon: new ServiceTree(),
+      marathon: {
+        tree: new ServiceTree()
```

Comment:
I simply named it tree as I wasn't happy with all other names that had come to my mind... I've changed it to `serviceTree` as `services` alone kinda looks like it's holding a flat list, hope that's ok for you.